### PR TITLE
Update rLogin

### PR DESCRIPTION
**Pull request by mariano-aguero**
No issue related ...

Update rlogin library, manage some breaking change

### CODE REVIEW

File: src/components/layouts/base/AppBar.vue [hunk 1]
```diff
@@ -160,10 +160,17 @@ export default {
...
+        const chainId = window?.ethereum?.chainId ?? 0
+
+        this.validateNetwork(chainId)
+
+        if (NETWORK_ID != parseInt(chainId)) {
+          throw new Error('Wrong network')
+        }
+
+        const rLoginResponse = await this.$rLogin.connect()
+        this.$provider = rLoginResponse.provider
```
Comment (mariano-aguero) [inline src/components/layouts/base/AppBar.vue:172]: The connect method does not throw an error when connecting to an invalid network, so we must validate it manually on the line L167 . Also return an object with the provider property

File: src/components/layouts/base/AppBar.vue [hunk 2]
```diff
@@ -185,10 +192,10 @@ export default {
         format.receipt.root = format.receipt.logsBloom
         Object.assign(this.$web3Provider.formatter, { format })
         Vue.web3Provider = this.$web3Provider
+        this.connectToWeb3()
       } catch (e) {
         this.$rLogin.clearCachedProvider()
       }
-      this.connectToWeb3()
```
Comment (mariano-aguero) [inline src/components/layouts/base/AppBar.vue:191]: I didn't know why this line was here, and not inside the try. I moved it to the line L195

**Review by fgregoribattista — DISMISSED** [review]
it's ok for me

**Comment by pmprete** [comment]
I told them about this error, in theory, they fixed it, and upgraded the version to @rsksmart/rlogin@1.0.2 https://www.npmjs.com/package/@rsksmart/rlogin

**Review by fgregoribattista — APPROVED** [review]
it's ok for me

**Review by galimba — APPROVED** [review]
LGTM

**Pull request merged**